src/runtime_src/core/edge/user/aie/aie.cpp

```diff
@@ -113,25 +115,30 @@ open_context(const xrt_core::device* device, xrt::aie::access_mode am)
 {
 #ifndef __AIESIM__
   auto drv = ZYNQ::shim::handleCheck(device->get_device_handle());
+  std::lock_guard access_lock_guard(am_lock);
+  if (access_mode != xrt::aie::access_mode::none) {
+	  return;
+  }
 
   auto current_am = drv->getAIEAccessMode();
   if (current_am != xrt::aie::access_mode::none)
     throw xrt_core::error(-EBUSY, "Can not change current AIE access mode");
 
   int ret = drv->openAIEContext(am);
   if (ret)
-      throw xrt_core::error(ret, "Fail to open AIE context");
+    throw xrt_core::error(ret, "Fail to open AIE context");
 
   drv->setAIEAccessMode(am);
   access_mode = am;
 #endif
 }
 
-bool
+xrt::aie::access_mode
 Aie::
-is_context_set()
+get_access_mode()
```

Comment:
Curious formatting.
Is this an output of clang-format?

Reply:
I just followed the same as used in this file before. I am not very sure about the formatting.

Reply:
Yes, if this is the local coding style.